game/game/gamescript.cpp

```diff
@@ -2793,6 +2826,12 @@ void GameScript::ai_turntonpc(std::shared_ptr<zenkit::INpc> selfRef, std::shared
     self->aiPush(AiQueue::aiTurnToNpc(npc));
   }
 
+void GameScript::ai_whirlaround(std::shared_ptr<zenkit::INpc> selfRef, std::shared_ptr<zenkit::INpc> npcRef) {
+  // This is supposed to be a fast turn to npcRef,
+  // for now, just also do a regular turn
```

Comment:
Why not implement it as `fast` turn right away?
PS: is my understanding correct, that "fast"=="immediate"?

Reply:
Probably me "just" not knowing enough about what to touch yet ;-) .

For the function I'm going with the description from https://github.com/Try/OpenGothic/wiki/Daedalus-scripts and https://www.worldofgothic.de/modifikation/index.php?go=g1functions ... both saying "fast turn to other".

Looking at the G1 script a great cinematic where it is used is with the Daemon in Xardas tower, where I also need to implement ai_turnaway as well :-) .

```
ai_turnaway(hero, self);
ai_output(hero, self, "Info_XardasDemon_INTRO_15_01");
ai_whirlaround(hero, self);
ai_output(hero, self, "Info_XardasDemon_INTRO_15_02");

```

I guess I'll dig a bit deeper into how that stuff works

Reply:
> I guess I'll dig a bit deeper into how that stuff works

Sounds good. If possible would be nice to have a video on how script above works in vanilla,as a reference.
As for sake of merging, `ai_whirlaround` can be decoupled to different PR

Reply:
I found a Let's play :-D ... https://youtu.be/GKpt0Y07gJw?t=1538 ... when talking to Xardas' daemon, which is this sequence from above

Reply:
I've actually managed to implement ai_whirlaround (+ai_turnaway) and moved both to a separate pull request.